sources/api/pluto/src/main.rs

```diff
@@ -383,3 +451,24 @@ mod test_pause_container_account {
         );
     }
 }
+#[cfg(test)]
+mod test_get_resource_to_reserve_in_range {
+    use crate::get_resource_to_reserve_in_range;
+    #[test]
+    fn get_resource_to_reserve_in_range_ok() {
```

Comment:
Test more inputs?

```
    fn get_resource_to_reserve_in_range_ok() {
```

Reply:
sure thing. sounds good to me

Reply:
My suggestion above to test `get_cpu_millicores_to_reserve` makes tests of `test_get_resource_to_reserve_in_range` less important. But it never hurts...